### PR TITLE
removed full class name in property docblock

### PR DESCRIPTION
**Pull request by jrushlow**

### CODE REVIEW

File: tests/FunctionalTests/Persistence/ResetPasswordRequestRepositoryTest.php [hunk 1]
```diff
@@ -10,6 +10,7 @@
 namespace SymfonyCasts\Bundle\ResetPassword\Tests\FunctionalTests\Persistence;
 
 use Doctrine\Bundle\DoctrineBundle\Registry;
+use Doctrine\Common\Persistence\ObjectManager;
```
Comment (weaverryan) [inline tests/FunctionalTests/Persistence/ResetPasswordRequestRepositoryTest.php]: Hmm, actually, this should use the `ObjectManager` from `doctrine/persistence` - this class is deprecated. I also see one more mention in `ResetPasswordTestFixtureRequestRepository.php`

**Comment by weaverryan** [comment]
This looks ready but tests are failing. Does it just need a rebase to get updated?

**Comment by jrushlow** [comment]
> This looks ready but tests are failing. Does it just need a rebase to get updated?

No, an upstream dependency changed and is now breaking the functional/integration tests. I have to dig in and see whats messing up the dir paths in our tests.

**Comment by jrushlow** [comment]
This PR was combined with #96 and is no longer relevant.

**Pull request closed without merging**